cmd/killgrave/main.go

```diff
@@ -41,14 +55,64 @@ func main() {
 		log.Println(err)
 	}
 
-	r := mux.NewRouter()
+	exit := make(chan struct{})
+	done := make(chan os.Signal, 1)
 
-	s := killgrave.NewServer(cfg.ImpostersPath, r)
-	if err := s.Build(); err != nil {
+	signal.Notify(done, os.Interrupt, syscall.SIGINT, syscall.SIGTERM)
```

Comment:
Is `os.Interrupt` really needed here? 🤔 

It seems (see [docs](https://golang.org/pkg/os/#Signal)) to be exactly the same as `syscall.SIGINT`, already used there.